io_scene_niftools/modules/nif_import/property/nodes_wrapper/__init__.py

```diff
@@ -76,7 +76,7 @@ def set_uv_map(self, b_texture_node, uv_index=0, reflective=False):
             self.tree.links.new(uv.outputs[6], b_texture_node.inputs[0])
         # use supplied UV maps for everything else, if present
         else:
-            uv_name = "TexCoordIndex" + str(uv_index)
+            uv_name = f"TexCoordIndex{uv_index}"
```

Comment:
Should we put an underscore between the prefix & index?

Reply:
why would we need it?

Reply:
Just to visually break up the prefix from the iterand.

Reply:
Added, see latest commit